pkg/apigw/registry/registry.go

```diff
@@ -74,7 +72,5 @@ func (r *Registry) Preload() {
 }
 
 func NewWorkflow() (wf filter.WfExecer) {
-	// implementation assumes that Corredor & Workflow options can not be changed
-	// in the run-time.
-	return service.Workflow(logger.Default().Named("workflow"), *options.Corredor(), *options.Workflow())
+	return service.DefaultWorkflow
```

Comment:
Should the system boot procedure push this into here?

Reply:
I don't see the benefit here, used the DefaultWorkflow as it handles the reloads (and wf caching) properly